### PR TITLE
SAR-4470: Added Overview view, controller and IT tests plus some refa…

### PR DESCRIPTION
**Pull request by r-melvin**
…ctoring

### CODE REVIEW

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/govuk_wrapper.scala.html [hunk 1]
```diff
@@ -36,13 +36,14 @@
...
     <meta name="format-detection" content="telephone=no" />
+    <link href=@controllers.routes.Assets.versioned("stylesheets/local-overrides.css") media="all" rel="stylesheet" type="text/css" />
 }
 
 @headerNavLinks = {}
 
 @insideHeader = {
     @uiLayouts.header_nav(
-      navTitle = Some("income-tax-subscription-eligibility-frontend"),
+      navTitle = Some("Use software to send Income Tax updates"),
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/govuk_wrapper.scala.html]: We have access to messages in this file, can this be moved into messages file along with a translation if available?

Comment (r-melvin) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/govuk_wrapper.scala.html]: I can't find a translation but I've made the change

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/overview.scala.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    <div class="grid-row interrupt-card">
+        <div class="column-two-thirds">
+
+            <h1>@messages("overview.heading")</h1>
+
+            <div class="form-group">
+                <p>@messages("overview.line_1")</p>
+                <p>@messages("overview.line_2")</p>
+                @Html(messages("overview.link_text", appConfig.guidanceUrl))
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/overview.scala.html]:
```suggestion
                <a id="guidance" href=@appConfig.guidanceUrl>@messages("overview.link_text")</a>
```
Can the link tag be taken out of the message?

File: conf/application.conf [hunk 3]
```diff
@@ -95,6 +95,10 @@ auditing {
   }
 }
 
+gov-uk {
```
Comment (AlexRimmerHMRC) [inline conf/application.conf]: Already added in line 113 - 115

**Pull request merged**